### PR TITLE
Add a description and files for logging in the stand alone environment

### PR DESCRIPTION
**Pull request by piste-jp**
### Explain the changes

1. Adding the configuration file of `rsyslog` for the standalone environment
2. Adding the configuration file of `logrotate` for the standalone environment
3. Updating the document how to build the stand alone environment to configure `rsyslog` and `logrotate` (Main target is RHEL8)

### Issues:

No issues to be linked at this time. This is an additional improvement of PR #7296.

### Testing Instructions:

Just adding the configuration file for stand alone environment and update the document for stand alone environment.

The configuration file is verified on my test bench.  

- [X] Doc added/updated

### CODE REVIEW

File: docs/standalone.md [hunk 1]
```diff
@@ -28,10 +28,45 @@ cd noobaa-core
 npm install
 npm run build
 # optional package everything into a single-executable at build/noobaa-core
-npm run pkg 
+npm run pkg
+sudo cp src/deploy/standalone/noobaa_configured_dns.conf /etc
```
Comment (guymguym) [inline docs/standalone.md:32]: this can be removed in favor of #7305

**Review by guymguym — APPROVED** [review]
@piste-jp-ibm Thanks! looks good. 
@liranmauda - just FYI.

File: docs/standalone.md (hunk 1 above)
Comment (liranmauda) [reply docs/standalone.md:32]: I guess also the empty src/deploy/standalone/noobaa_configured_dns.conf can be removed also

Comment (piste-jp-ibm) [reply docs/standalone.md]: Thanks. I removed it from both repository and the doc.

**Comment by piste-jp-ibm** [comment]
I rebased to the master and removed `noobaa_configured_dns.conf`.

**Comment by liranmauda** [comment]
Please rebase and squash all the commits into a single commit.

**Comment by piste-jp-ibm** [comment]
@liranmauda,
 
I tried to squash my commits. Do I need to fix branch point [bff1022] also?

**Review by liranmauda — APPROVED** [review]

**Comment by liranmauda** [comment]
@piste-jp-ibm Thanks! 
once it passes the tests I will merge it.

**Pull request merged**